src/matching/Generic_vs_generic.mli

```diff
@@ -22,3 +22,11 @@ val m_any : AST_generic.any Matching_generic.matcher
 
 val hook_find_possible_parents :
   (AST_generic.dotted_ident -> AST_generic.name list) option ref
+
+val m_compatible_type :
```

Comment:
you can add a comment like (* used now for metavariable-type: in Match_search_mode.ml *)